README.md

```diff
@@ -1,44 +1,6 @@
-# CIbyl
+CIbyl
+-----
 
-Command line interface for querying CI environments
-
-## Installation
-
-```
-pip install cibyl
-```
-
-Set up configuration in one of the following paths:
-  * ~/.cibyl/cibyl.yaml
-  * /etc/cibyl/cibyl.yaml
-
-A valid configuration should specify an environment, its system(s) and the
-system(s) details.
-Use the configuration below as a minimal configuration file.
-```
-environments:
-    env_1:
-        jenkins_system:
-            system_type: ""
-```
-For query purposes, add sources in the following format:
-
-```
-environments:
-  staging:
-    environment1:
-      system_type: 'SYSTEM_TYPE'
-      sources:
-        source1:
-          driver: 'DRIVER'
-          username: 'USERNAME'
-          token: 'PERSONAL_TOKEN'
-          url: 'https://URL.com'
-```
-## Usage
-
-To list existing environments and their systems, run `cibyl`
-
-## How to Contribute?
-
-Contributions are done via GitHub Pull Requests
+Official Documentation
+----------------------
+For more information please read our `Documentation <http://cibyl.readthedocs.org>`_
```

Comment:
In addition to the link to the docs, I  would keep the info that we had before as a quick glance for the user. I usually find  the github of a project before its docs and I think it's nice to be able to get some vague idea of the project from the readme

Reply:
100% on what Joan wrote.
I really like to have quick glance in README.md on what the project is about + quick start (installation, perhaps basic usage, ...)